api_tests/src/ledgers/ledger_lock.ts

```diff
@@ -16,7 +16,7 @@ export default async function ledgerLock(
         retries: {
             retries: 10,
             minTimeout: 200,
-            maxTimeout: 2000,
```

Comment:
This should only be the max timeout _per iteration_ and we do 10 retries :o

Reply:
Oops.